diff --git arduino-compat/arduino-pico-extra/SPI.h arduino-compat/arduino-pico-extra/SPI.h
index bf08f183..4e3e26f2 100644
--- arduino-compat/arduino-pico-extra/SPI.h
+++ arduino-compat/arduino-pico-extra/SPI.h
@@ -1,4 +1,4 @@
 #pragma once
-#include "../arduino-pico/libraries/SPI/SPI.h"
+#include "arduino-pico/libraries/SPI/SPI.h"
 #undef SPI_HAS_TRANSACTION
 #define SPI_HAS_TRANSACTION 1
diff --git arduino-compat/arduino-pico-extra/pins_arduino.h arduino-compat/arduino-pico-extra/pins_arduino.h
index b3ddff49..116adf06 100644
--- arduino-compat/arduino-pico-extra/pins_arduino.h
+++ arduino-compat/arduino-pico-extra/pins_arduino.h
@@ -36,4 +36,4 @@
 #define SPI_HOWMANY    (2u)
 #define WIRE_HOWMANY   (2u)
 
-#include "../arduino-pico/variants/generic/common.h"
+#include "arduino-pico/variants/generic/common.h"
